plugin-gradle/README.md

```diff
@@ -139,10 +139,10 @@ To apply freshmark to all of the `.md` files in your project, with all of your p
 ```gradle
 spotless {
 	freshmark {
-		target 'README.md', 'CONTRIBUTING.md'	// defaults to '**/*.md'
-		propertiesFile('gradle.properties')		// loads all the properties in the given file
+		target 'README.md', 'CONTRIBUTING.md'			// defaults to '**/*.md'
+		propertiesFile 'fresh.xml', 'mark.properties'	// loads all the properties in the given files
 		properties {
-			it.put('key', 'value')				// specify other properties manually
+			it.put('key', 'value')						// specify other properties manually
```

Comment:
Out spotless configuration *enforces* that leading indentation in [Markdown files must be tabs](https://github.com/diffplug/spotless/blob/32de1fd58e5d0520febf637760be916a6ad71816/spotlessSelf.gradle#L32-L37).

The point of Spotless is so that open source projects can take contributions without arguing over style.  Let's live out our mission and focus on content!  If the code style passes our style checks, it meets the style.  These lines LGTM.